docs/getting-started.md

```diff
@@ -199,19 +199,6 @@ ID              Version Name
 esp8266:esp8266 2.5.2   esp8266
 ```
 
-Alternatively, you can pass a link to the additional package index file with the
-`--additional-urls` option, that has to be specified every time and for every
-command that operates on a 3rd party platform core, for example:
-
-```sh
-$ arduino-cli  core update-index --additional-urls https://arduino.esp8266.com/stable/package_esp8266com_index.json
-Updating index: package_esp8266com_index.json downloaded
-
-$ arduino-cli core search esp8266 --additional-urls https://arduino.esp8266.com/stable/package_esp8266com_index.json
-ID              Version Name
-esp8266:esp8266 2.5.2   esp8266
-```
-
```

Comment:
@per1234 do we have another place in our docs where the usage of `--additional-urls` is described in addition to this one that you are removing?
IIRC we wrote this paragraph because users got confused by this option. I'm ok regarding this PR, but I would like to keep info regarding this option in a more suited place of the docs.

Reply:
The only other place it's described is on every page of the command reference (e,.g., https://arduino.github.io/arduino-cli/dev/commands/arduino-cli/#options), which doesn't give much information on it. I don't see another place in the documentation where this information would fit.

I'm fine with leaving it in the Getting Started page. I explained my reasoning in [the commit message](https://github.com/arduino/arduino-cli/pull/782/commits/67b94ba4208c759e61c80b412430eb12d0e363b4), but I don't have super strong feelings about this subject.

Reply:
The [FAQ Page](https://arduino.github.io/arduino-cli/latest/FAQ/) maybe could be a good place to move this whole paragraph, for example in a "how the --additional-urls option works?" 
WDYT?

Reply:
I've given it some thought and I just don't think it's a good fit for the FAQ page.

What I suggest is that I abandon my proposal to remove  the `--additional-urls` information from the Getting Started page (making this PR solely about the "Clarify wording of upload instructions in Getting Started page" proposal). 

I can perhaps revisit the idea of moving the `--additional-urls` information from the Getting Started page if we someday add another page to the documentation that is appropriate for it (something like a "Going further" page).